packages/backend/package.json

```diff
@@ -58,7 +58,6 @@
     "apollo-server-koa": "^3.11.1",
     "axios": "0.26.1",
     "base64url": "^3.0.1",
-    "bcrypt": "^5.0.1",
```

Comment:
Can these other dependencies also be removed?
https://github.com/interledger/rafiki/commit/b7570da43bdb7222cf78d37f9261934fcbf232b1#diff-e095a6dd10bb40447d8290e79f70c15334e874e757e25a569b9d3449ed4fe27dR33-R56